src/main/java/org/fineract/messagegateway/sms/api/SmsApiResource.java

```diff
@@ -50,7 +55,18 @@ public SmsApiResource(final SMSMessageService smsMessageService) {
     public ResponseEntity<Void> sendShortMessages(@RequestHeader(MessageGatewayConstants.TENANT_IDENTIFIER_HEADER) final String tenantId,
     		@RequestHeader(MessageGatewayConstants.TENANT_APPKEY_HEADER) final String appKey, 
     		@RequestBody final List<SMSMessage> payload) {
-    	this.smsMessageService.sendShortMessage(tenantId, appKey, payload);
+        
+    	WirepickSMS sms = new WirepickSMS() ;
+         for(SMSMessage smessage : payload) {
```

Comment:
Try to follow camel case coding style while naming variables.
```suggestion
         for(SMSMessage sMessage : payload) {
```